src/components/ChatLog.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import './ChatLog.css';
+// import chatMessages from './data/messages.json';
+
+const ChatLog = (props) => {
+  const msgComponents = [];
+  let keyCounter = 0;
+  for (const msg of props.entries) {
+    msgComponents.push(
+      <li key={keyCounter}>
```

Comment:
The `li` element should not be a child of a `div` element. They should only be the children of `ul`, `ol`, or `menu` elements. The `li` element here is not needed.